docs/overview_and_guide.rst

```diff
@@ -71,7 +71,7 @@ Linux
 Additional dependencies for development installations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- You need to make sure that your system has ``solc``, the ethereum solidity compiler installed. Refer to `its documentation`_ for the installation steps.
+- You need to make sure that your system has ``solc``, the ethereum solidity compiler installed. Refer to `its documentation`_ for the installation steps. (Note: It is possible to use Raiden without solc because every official realease brings precompiled smartcontracts)
```

Comment:
I think this is superfluous since it says "for development installations" in the title above. If you use the binaries you don't need it, but this part is not about the binaries.